core/domain/platform_feature_services.py

```diff
@@ -321,3 +325,25 @@ def get_platform_parameter_value(
     context = _create_evaluation_context_for_server()
     param = registry.Registry.get_platform_parameter(parameter_name)
     return param.evaluate(context)
+
+
+def feature_flag_valid_on_current_server(feature_stage: str) -> bool:
```

Comment:
This is a function so its name should start with a verb.

Reply:
No longer valid as the logic has been moved.